### PR TITLE
SAK-47721 rubrics: use uniqe ids for rubrics summary panels

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-47721

### CODE REVIEW

**Comment by adrianfish** [comment]
@davidpbauer Could you take a look at this, give it a review, please? I know you're not a reviewer at present, for whatever reason, but if you just put a comment in, that'll do. Cheers!

File: webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ export class SakaiRubricGrading extends rubricsApiMixin(RubricsElement) {
     this.criteria = [];
     this.totalPoints = 0;
 
+    this.uniqueKey = Math.floor(Math.random() * 100);
```
Comment (davidpbauer) [inline webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js:21]: While unlikely to repeat in this case, it doesn't guarantee uniqueness. I've seen `Math.floor(Date.now() * Math.random())` used for extra comfort.

**Comment by davidpbauer** [comment]
The unique ids make sense and I think your changes look fine. I did add a comment about ensuring the id has a unique number appended at the end using new date instead of 100 ... but honestly it just feels like overkill in this case.

Leah Bergman on our team took a look at this locally and compared to Nightly 23.x and the behavior we see locally is different than what is being seen on Nightly. It's possible something was missed when we ported it over or to the new version.
Here's a summary of the behavior she has seen https://docs.google.com/document/d/1GPAvoLwsBcF5W-n0Gtg0kd8mir8jmzkTr5h4DM5_hBk/edit?usp=sharing

**Comment by adrianfish** [comment]
@davidpbauer I've changed the uniqueKey to be instanceSalt and used your Date.now suggestion. It only has to be unique on the page, but still, I like that date.now idea.

**Comment by adrianfish** [comment]
Thanks for the review, btw. The diff between what you have at Dayton and what's on master, that should go in another ticket. I did a lot of the massaging work, so I may well have messed something up.

**Pull request merged**